src/main.ts

```diff
@@ -24,6 +24,25 @@ export class Main extends Command {
     await this.config.runCommand(id, argv)
   }
 
+  splitArgv() {
+    // For example, if this.argv is ['user', 'add', 'Peter'] and this.config.commandIDs
+    // contain the 'user add' command, this function returns {id: 'user add', argv: ['Peter']}
+    let argvIndex = 0
+    let id = ''
+    let idCandidate = this.argv[argvIndex]
+
+    while (this.config.commandIDs.includes(idCandidate)) {
```

Comment:
I just realized that this getter is pretty expensive, it should be called once, not repeatedly. (I'd also argue that `commandIDs` shouldn't be a getter in the first place but that would be a larger refactoring I guess.)

Reply:
Resolved in https://github.com/oclif/command/pull/51/commits/4aa3dfeb642cbd9d27a02493fc8c107a818172fd.